tests/__init__.py

```
@@ -127,12 +104,18 @@
 		test_hear_cleanup,
-		test_fuzz,
 		test_hear_data_processing,
 		test_exceptions,
 		test_hear_keyboard_interrupt,
 		test_hear_server
 	]
+
+	try:
+		from tests import test_fuzz
+		depends.insert(10, test_fuzz)
+	except Exception as e:  # pragma: no branch
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)